test/SyntaxTest.re

```diff
@@ -37,19 +36,14 @@ describe("Syntax", ({describe, _}) => {
 
       let errorRange = List.hd(errors);
 
-      let startLine = errorRange.startPosition.line;
-      let endLine = errorRange.endPosition.line;
-
-      let startColumn = errorRange.startPosition.column;
-      let endColumn = errorRange.endPosition.column;
+      expect.int((errorRange.start.line :> int)).toBe(0);
+      expect.int((errorRange.stop.line :> int)).toBe(0);
 
-      expect.int(startLine).toBe(0);
-      expect.int(endLine).toBe(0);
-
-      expect.int(startColumn).toBe(2);
-      expect.int(endColumn).toBe(3);
+      expect.int((errorRange.start.column :> int)).toBe(2);
```

Comment:
Interesting, I hadn't seen the coercion operator before `:>` or the `type t = pri ...` before, used in `editor-core-types`

Reply:
I used it for `NormalizedTime` in Revery too. Outside of tests it's probably better to be explicit though.